### PR TITLE
Change Marker Color

### PR DESCRIPTION
**Pull request by KamilMa**

### CODE REVIEW

File: client/src/components/Map/Map.js [hunk 1]
```diff
@@ -45,11 +45,11 @@ export default class Map extends Component {
         </Link>
       </CommonMobileLink>
       <ReactMapGl
-      mapboxApiAccessToken={ACCESS_TOKEN}
-      mapStyle={MAP_STYLE}
-      maxZoom={16}
-      {...this.state.viewport}
-      onViewportChange={(viewport) => this.setState({ viewport })}>
+        mapboxApiAccessToken={ACCESS_TOKEN}
```
Comment (tkleszczewski) [inline client/src/components/Map/Map.js]: Why add space / tab ?

File: client/src/components/Map/Map.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ export default class Map extends Component {
                 key={place.id}
                 offsetTop={-40}
                 offsetLeft={-15}>
-                <FontAwesomeIcon style={{color: colors.burntSienna}} icon="map-marker-alt" size="4x"></FontAwesomeIcon>
+                <FontAwesomeIcon style={{ color: colors.sulu }} icon="map-marker-alt" size="4x"></FontAwesomeIcon>
```
Comment (tkleszczewski) [inline client/src/components/Map/Map.js]: Why added spaces between braces and object props

File: client/src/components/Map/Map.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ const MapContainer = styled.section`
 `;
 
 export default class Map extends Component {
-  constructor (props) {
```
Comment (tkleszczewski) [inline client/src/components/Map/Map.js]: why removed space between name and parentheses ?

File: client/src/components/Map/Map.js (hunk 2 above)
Comment (KamilMa) [reply client/src/components/Map/Map.js]: where?

Comment (KamilMa) [reply client/src/components/Map/Map.js]: my prvsc ogram has such settings

File: client/src/components/Map/Map.js (hunk 3 above)
Comment (KamilMa) [reply client/src/components/Map/Map.js]: my vsc has such settings.

File: client/src/components/Map/Map.js (hunk 1 above)
Comment (KamilMa) [reply client/src/components/Map/Map.js]: My vsc has such settings.

**Pull request merged**